db/init.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE Users (
+	id SERIAL PRIMARY KEY,
+	login VARCHAR(30) UNIQUE,
+	password VARCHAR(30),
+	age INTEGER
+);
+
+INSERT INTO Users (login, password, age) VALUES 
+('JohnDoe12', 'pasw123', 20), 
+('JaneDoe', 'pasw321', 19), 
+('JohnSmith112', '123pasw', 63), 
+('JaneSmith221', '321pasw', 25);
+
+CREATE TABLE Groups (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR(30) UNIQUE,
+	permissions VARCHAR(20)[]
+);
+
+INSERT INTO Groups (name, permissions) VALUES
+('ADMIN', '{READ, WRITE, DELETE, SHARE, UPLOAD_FILES}'),
+('USER', '{READ, WRITE, SHARE}');
+
+
+CREATE TABLE UserGroup (
```

Comment:
YES :-) do all work as close to where it is needed as possible - using cascade :-) nice